### ISSUE TITLE
There's no man-page

### ISSUE THREAD

**Issue by Tymewalk** [Tymewalk/Tymeventure#3]
Just something I planned on doing for a while.

If anyone has tips on how to make one through `setup.py` (as in the man-page is also build through `setup.py`), please leave a note in this issue thread.

**Comment by DrKat123** [comment]
0_0
"Man" page?
Moar liek main page

**Comment by Tymewalk** [comment]
@DrKat123 No, man-page, as in the console command `man` (short for manual).

**Comment by DrKat123** [comment]
Oh...manual
